bin/build.sh

```diff
@@ -362,6 +365,24 @@ then
   puts "Built Corral"
 fi
 
+if [ ${BUILD_SYMBOOGLIX} -eq 1 ]
+then
+  puts "Building Symbooglix"
+
+  cd ${ROOT}
+  git clone https://github.com/symbooglix/symbooglix.git ${SYMBOOGLIX_DIR} 
```

Comment:
I tried making this change, but the build failed, printing "Cannot locate a solution file." Something having to do with nuget.exe I think.